proxy/proxies/nginx/nginx.go

```diff
@@ -50,10 +52,12 @@ import (
 )
 
 const (
-	defaultNginxIngressImage = "cargo.caicloud.io/caicloud/ingress-nginx:v0.1.0"
+	defaultNginxIngressImage = "cargo.caicloud.io/caicloud/nginx-ingress-controller:0.9.0-beta.8"
```

Comment:
An env or cmd for image path is better than a specified default image.

If user did not specify a image, just fatal with an error info.